### PR TITLE
Fixed bug with an undefined variable.

### PR DESCRIPTION
**Pull request by stereomon**
Fixed bug with an undefined variable.

### CODE REVIEW

**Review by dereuromark — APPROVED** [review]
You can directly merge this.

**Comment by dereuromark** [comment]
Side note for defensive coding:
realpath() can also return false if for some reason path doesnt work out.
We might want to follow up and throw meaningful exception or return early in that case rather than passing false into a method, that expects a valid path string.
It currently would blow up without meaningful context as message.

We can also do

    realpath($dirname . '/.git/hooks') ?: null

and make input nullable for the method, including null then for invalid path response.

PHPStan should also confirm this requirement by the way.

File: src/GitHook/Composer/Scripts/HookInstaller.php [hunk 1]
```diff
@@ -176,7 +176,7 @@ public static function installEcoHooks(Event $event)
             $src = realpath($hookDirectory . DIRECTORY_SEPARATOR . $hook);
             foreach ($modulesDirs as $dirname) {
                 $destinationDirectory = realpath($dirname . '/.git/hooks');
-                if (self::checkDirectoryPermissions($gitHookDirectory, $event)) {
+                if (self::checkDirectoryPermissions($destinationDirectory, $event)) {
```
Comment (matweew) [inline src/GitHook/Composer/Scripts/HookInstaller.php:179]: static:: ?

Comment (dereuromark) [reply src/GitHook/Composer/Scripts/HookInstaller.php:179]: No big deal for this kind of repo.
This is project level, not framework level.

**Pull request merged**